packages/containerd/0001-reduce-logging-when-no-errors-have-occurred.patch

```diff
@@ -0,0 +1,63 @@
+From c3b91def7e96d11815e681eb2c36b5d89a21119d Mon Sep 17 00:00:00 2001
+From: Matt Briggs <brigmatt@amazon.com>
+Date: Wed, 1 Apr 2020 16:55:04 -0700
+Subject: [PATCH] reduce logging when no errors have occurred
+
+---
+ .../containerd/cri/pkg/server/container_execsync.go           | 4 +++-
+ .../containerd/cri/pkg/server/instrumented_service.go         | 4 ++--
+ vendor/github.com/containerd/cri/pkg/server/io/exec_io.go     | 2 +-
+ 3 files changed, 6 insertions(+), 4 deletions(-)
+
+diff --git a/vendor/github.com/containerd/cri/pkg/server/container_execsync.go b/vendor/github.com/containerd/cri/pkg/server/container_execsync.go
+index b46e6e56..55629ca2 100644
+--- a/vendor/github.com/containerd/cri/pkg/server/container_execsync.go
++++ b/vendor/github.com/containerd/cri/pkg/server/container_execsync.go
+@@ -186,9 +186,11 @@ func (c *criService) execInContainer(ctx context.Context, id string, opts execOp
+ 		return nil, errors.Wrapf(execCtx.Err(), "timeout %v exceeded", opts.timeout)
+ 	case exitRes := <-exitCh:
+ 		code, _, err := exitRes.Result()
+-		log.G(ctx).Infof("Exec process %q exits with exit code %d and error %v", execID, code, err)
+ 		if err != nil {
++			log.G(ctx).Errorf("Exec process %q exits with exit code %d and error %v", execID, code, err)
+ 			return nil, errors.Wrapf(err, "failed while waiting for exec %q", execID)
++		} else {
++			log.G(ctx).Debugf("Exec process %q exits with exit code %d and error %v", execID, code, err)
```

Comment:
You don't need this inside `else` and can safely out-dent it.  The error condition is already handled by the `return` and control flow won't come back out if there is an error.

Reply:
Damn, you're right. I'll make that change.